lochness/mindlamp/__init__.py

```diff
@@ -99,49 +101,71 @@ def sync(Lochness: 'lochness.config',
     # set the cut off point as UTC 00:00:00.00
     ct_utc_00 = ct_utc.replace(hour=0, minute=0, second=0, microsecond=0)
 
-    for days_from_ct in range(days_to_check):
-        # n day before current time
+    # Extra information for future version
+    # study_id, study_name = get_study_lamp(LAMP)
+    # subject_ids = get_participants_lamp(LAMP, study_id)
+    subject_id = subject.mindlamp[f'mindlamp.{subject.study}'][0]
+
+    # set destination folder
+    dst_folder = tree.get('mindlamp',
+                          subject.protected_folder,
+                          processed=False,
+                          BIDS=Lochness['BIDS'])
```

Comment:
Bringing `subject_id` and `dst_folder` out of the `for` loop is a good optimization.